README.md

```diff
@@ -138,19 +138,9 @@ Subsequently, we defined stopwords and stopwords were cleaned and removed thereb
 
 At last, we then performed Stemming(reducing the words to their derived stems) and Lemmatization(reducing the derived words to their root form known as lemma) for better results.
 
-### Proposed ML Models
```

Comment:
I think we should leave some type of history on the decision-making for our ML Model within this README.
I will approve for now, but will update this section.